public/_posts/_articles/yearn-finance/yvboost/es.md

```diff
@@ -0,0 +1,99 @@
+---
+title:  "Más impulsos, compuestos y mojitos para ti con yvBOOST"
+image:
+  src: ./cover.jpg
+  width: 1050
+  height: 543
+date: '2021-09-14'
+author: Qosmonot
+translator: RestlessMik3
+---
+
+# Más impulsos, compuestos y mojitos para ti con yvBOOST
+
+### La relación de colaboración de Yearn con [Curve.fi](http://curve.fi/) es fundamental para nuestra infraestructura y fundamental para impulsar tus impulsos.
+
+🔵🤝🌈
+
+_publicación escrita por_ [_qosmonot_](http://twitter.com/qosmonot), un colaborador de Yearn
+
+Muchas vaults de Yearn tienen estrategias basadas en los grupos de Curve y, por lo tanto, usan el token CRV para optimizarlas votando para dirigir las asignaciones de recompensas hacia esos grupos. Yearn ha estado aumentando constantemente su titularidad de tokens CRV para proporcionar recompensas cada vez mayores para esas vaults.
```

Comment:
AS-IS: Muchas vaults de Yearn tienen estrategias basadas en los grupos de Curve y, por lo tanto, usan el token CRV para optimizarlas votando para dirigir las asignaciones de recompensas hacia esos grupos. Yearn ha estado aumentando constantemente su titularidad de tokens CRV para proporcionar recompensas cada vez mayores para esas vaults.

TO-BE: Muchas vaults de Yearn tienen estrategias basadas en los fondos de Curve y, por lo tanto, usan el token CRV para optimizarlas votando para dirigir las asignaciones de recompensas hacia esos fondos. Yearn ha estado aumentando constantemente su titularidad de tokens CRV para proporcionar recompensas cada vez mayores para esas vaults.

Reply:
Switching "groups" by "fondos" I think makes more sense. I do struggle in general to translate "pools" in this context though, so open to other suggestions.

Reply:
ok, done